python/img_hdf5_reader.py

```diff
@@ -107,6 +108,58 @@ def to_pngs(self, channel_clips, sample, json_file):
         # This JSON file is not used right now:
         # really just a list of the files processed.
 
+    def to_ometiff(self, channel, sample, ometif_path, clip, ometif_basename):
+        MAX_ALLOWED = 256
+        NP_TYPE = np.int8
+
+        scaled_sample = self.scale_sample(
+            channel=channel,
+            sample=sample,
+            max_allowed=MAX_ALLOWED,
+            clip=clip
+        ).astype(NP_TYPE)
+        return scaled_sample
+
+    def to_ometiffs(self, channel_clips, sample, json_file):
+        channels = {}
+        s3_target = open('s3_target.txt').read().strip()
+        images = []
+        for (channel, clip) in channel_clips:
+            channels[channel] = {
```

Comment:
You only end up using the keys: Just a set, instead?

(I think you got here by copy-and pasting the PNG code: I think you either need to refactor this so to pull out the the shared code between the two... or you need to edit this down so you're only doing the work that actually needs to be done.)

Reply:
(So, for example, I don't think the s3_target is actually used, in the end.)